pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/ecdhes.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package ecdhes provides implementations of JWE encryption using ECDH-ES KW primitives.
```

Comment:
JWE is not, itself, a crypto primitive. JWE belongs at a higher layer. Again, please clarify the intended meaning.

Reply:
@Baha-sk I think we should, at least, have a TODO to refactor JWE specifics out where possible.

Reply:
see my previous comment

Reply:
status?

Reply:
JWE reference is now removed from this change. It will be introduced in a another PR specifically for JWE outside of crypt package.

Reply:
created #1638 to build and parse the JWE message (outside of Crypto package)